courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<p>(personal preference) remove 1st paragraph, second paragraph suffices. Though fine how it is. Or maybe just remove "in this lab" from 2nd paragraph. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Ya, once these are all done I want to do another pass on these intros. I was originally following a template, but it is redundant I think. Then again, do students even read this section haha?</p>

Reply:
ok